channels-rke2.yaml

```diff
@@ -630,6 +630,15 @@ releases:
         version: v3.24.1-build2022101102
     featureVersions: &featureVersions-v1
       <<: *featureVersions-v1-22-11-rke2r1
+  - version: v1.22.16-rc1+rke2r1
+    minChannelServerVersion: v2.6.3-alpha1
+    maxChannelServerVersion: v2.6.99
+    serverArgs: *serverArgs-v1-22-11-rke2r1
+    agentArgs: *agentArgs-v1-22-11-rke2r1
+    charts: &charts-v1-22-16-rc1-rke2r1
+      <<: *charts-v1-22-15-rke2r2
+    featureVersions: &featureVersions-v1
```

Comment:
This appears to be redefining the `featureVersions-v1` anchor; is this intentional? We usually have just one definition of any given anchor, and then multiple uses of it from merge keys.